app/helpers/organisations_helper.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+module OrganisationsHelper
+  def organisation_logo(organisation)
+    if organisation.name == 'Department for Work and Pensions'
+      render partial: 'shared/dwp_logo', locals: { organisation: }, class: 'govuk-image',
+             alt: 'Department for Work and Pensions'
```

Comment:
Do we need the alt property here? I noticed we've also got it set in the partial.

Reply:
Just noticed that the alt tags aren't working on the partial so maybe you can't set it on the GOV UK logo component. Perhaps because it doesn't render an <img> tag, only <a> tags and <span> tags?

Reply:
Oh I see, it's setting the image as background in the CSS. In that case yes, no alt text necessary!